pump/enum_file_writer.go

```diff
@@ -0,0 +1,71 @@
+package pump
```

Comment:
This is not an enumerator (the thing that tells you what to copy) so this should not be named like that.

You could also put those writers in a folder to namespace them and have something simpler/cleaner:
- `errorwriter.File`
- `errorwriter.Null`

Reply:
And then you would have two code files, and no need to intertwine them like in this file.

Reply:
Hmm I see your point. What would be a good name to explain that it's the writer generating THE `enum file`?

I could put them to a directory as well but I followed this flat structure given we have no dirs, didn't want to introduce a new package for just this scenario.

![Screenshot from 2021-03-22 18-33-08](https://user-images.githubusercontent.com/3595242/112033786-1581e680-8b3e-11eb-9a10-ca5b40d84e25.png)

Reply:
there was no other directory because there was no need for them ... until now :)